client/packages/common/src/intl/strings/IntlStrings.ts

```diff
@@ -38,3 +38,8 @@ export const useTranslation = (ns?: Namespace): TypedTFunction<LocaleKey> => {
     [t]
   );
 };
+
+export const translationExistsInLocale = (key: string): boolean => {
```

Comment:
Label in columns is now a string, so we want to check if the translation exists in the translation file first so determine if we should use the string or translation